meta/classes/create-spdx.bbclass

```diff
@@ -907,8 +907,15 @@ def combine_spdx(d, rootfs_name, rootfs_deploydir, rootfs_spdxid, packages):
     doc.packages.append(image)
 
     for name in sorted(packages.keys()):
-        pkg_spdx_path = deploy_dir_spdx / "packages" / (name + ".spdx.json")
-        pkg_doc, pkg_doc_sha1 = oe.sbom.read_doc(pkg_spdx_path)
+        try:
+            pkg_spdx_path = deploy_dir_spdx / "packages" / (name + ".spdx.json")
+            pkg_doc, pkg_doc_sha1 = oe.sbom.read_doc(pkg_spdx_path)
+        except FileNotFoundError:
+            if (d.getVar('BUILD_IMAGES_FROM_FEEDS') or "") == "1":
```

Comment:
Why `or ""`?

Reply:
I saw this done other places with other variables. I believe it's to ensure that there is some value but I'm not sure. @amstewart - Any idea why this pattern is used throughout the code or if it's necessary here?

Reply:
That safety is useful when you're passing the returned string into a subsequent method which may not handle `NoneType` inputs well. If you grep through the OE-core code, it's mostly used in places like `(getVar(...) or "").split()` and `(getVar(...) or "").strip()`.

I don't think it should be necessary here. So I would recommend we remove it for cleanliness.

Reply:
Fixed.